app/javascript/mastodon/reducers/compose.js

```diff
@@ -54,6 +54,7 @@ const initialState = ImmutableMap({
   default_sensitive: false,
   resetFileKey: Math.floor((Math.random() * 0x10000)),
   idempotencyKey: null,
+  last: null,
```

Comment:
I'm not sure about having this information here, it seems to be only ever useful to the particular case of the web intent page… there ought to be a better way, although I'm not familiar enough with React to know better :/

Reply:
I don't think there is a better way for this particular case. There are several options such as to have different Redux store or to utilize callbacks and React state. However they bring too much complexity for just one variable.